_ml-commons-plugin/model-serving-framework.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: default
 title: Model-serving framework 
-has_children: false
+has_children: true
 nav_order: 110
 ---
 
 # Model-serving framework
 
-The Model-serving framework is an experimental feature. For updates on the progress of the Neural Search plugin, or if you want to leave feedback that could help improve the feature, join the discussion in the [Model-serving framework forum](https://forum.opensearch.org/t/feedback-machine-learning-model-serving-framework-experimental-release/11439).    
+The model-serving framework is an experimental feature. For updates on the progress of the Neural Search plugin, or if you want to leave feedback that could help improve the feature, join the discussion in the [Model-serving framework forum](https://forum.opensearch.org/t/feedback-machine-learning-model-serving-framework-experimental-release/11439).    
```

Comment:
Not for this line. As I can't comment on line 26, `### Model format`, so just add comment here.

From 2.5, we can support ONNX model too. Refer to https://onnx.ai/

Reply:
Going to address this in a separate PR.